validate/validate.go

```diff
@@ -519,6 +520,12 @@ func (v *Validator) CheckLinux() (msgs []string) {
 		}
 	}
 
+	if v.spec.Linux.MountLabel != "" {
+		if err := label.Validate(v.spec.Linux.MountLabel); err != nil {
```

Comment:
> It just checks whether label contains unexpected options.

But there is nothing in the spec about “MUST be a valid SELinux label according to [some SELinux spec]” or “MUST NOT contain `z` or `Z`” or anything like that.  Currently spec-readers have to make that jump on their own.

And I was suggesting host-specific because even if `z` and `Z` are not legal labels for today's kernel, are we sure that they will be illegal labels for *all* kernels?  Maybe someone compiles a Linux kernel where they are legal, and the spec's kernel-punt-where-possible approach is designed to support that sort of thing.